infrastructure/cache-iam-policies.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
This LGTM, we can add Karpenter in here as well in a follow up. 

Can you update the README so that someone know how to perform a version update?

Reply:
Done, added note to README